.travis.yml

```diff
@@ -5,8 +5,21 @@ python:
     - '3.6'
 install:
     - make install-dev
-    - pip install codecov
+    - pip install codecov httpie
+    - wget -O oc.tar.gz https://github.com/openshift/origin/releases/download/v3.7.2/openshift-origin-client-tools-v3.7.2-282e43f-linux-64bit.tar.gz
+    - tar -zxvf oc.tar.gz
+    - cp openshift-origin-client-tools-v3.7.2-282e43f-linux-64bit/oc .
```

Comment:
Similar comment to below. maybe we could make this a Make command in integraes' Makefile.

My thought is it will be instantly reusable if it is in the Makefile

Reply:
This type of install is very linux specific, in addition, on user machines we'd probably be placing this in an actual bin folder so it'd be part of the path.